### ISSUE TITLE
ebay-video: play button barely visible

### ISSUE THREAD

**Issue by darkwebdev** [eBay/ebayui-core#1881]
# Bug Report

## Description
`Play` button has very low contrast

## Screenshots
Before Skin 16:
<img width="181" alt="image" src="https://user-images.githubusercontent.com/2737310/231731050-8549c779-ef68-4145-baa4-e592d45d659c.png">

After:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/2737310/231731170-9ff80706-8b42-479a-855a-3f8f6b8b4dd5.png">

**Comment by ArtBlue** [comment]
I don't think this is an ebay ui core bug. If anything, it's arguably an issue in Skin, but it's probably not that either. We simply need a more bulletproof icon from DS to work well in both use cases - dark background and light.

**Comment by agliga** [comment]
The icon should be the play filled one. 
Although i see the icon show up as blue and not like in this image. It fits better

**Comment by agliga** [comment]
Spoke to Ryan, it should be `play-64-colored`

**Comment by darkwebdev** [comment]
Thanks @agliga !
